examples/flutter_gallery/lib/welcome/home.dart

```diff
@@ -0,0 +1,151 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:flutter_gallery/welcome/steps/playground_step.dart';
+
+import 'step.dart';
+import 'steps/all.dart';
+
+const Color _kWelcomeBlue = Color(0xFF0175c2);
+const int _kAutoProgressSeconds = 6;
```

Comment:
Is this the constant for the delay between each of the pages in the onboarding experience? If so, I think we should raise this to 10 seconds. I struggle with reading all the text and interpreting the imagery in the 6 seconds.

Reply:
Increased to 10 sec